Controls/Data.UWP/Data/Engine/ItemPlacement.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Telerik.Data.Core
+{
+    public enum ItemPlacement
```

Comment:
We need comments on these.
Do we need public enum for this? Can we pass bool argument - true (for before), false otherwise?

Reply:
What would Placement true / false mean?

Reply:
If we decide to preserve it a documentation should be added or the StyleCop will fail. 😄